shared/desktop/renderer/remote-tracker.js

```diff
@@ -30,6 +32,7 @@ class RemoteTracker extends Component<void, Props, void> {
       <div>
         {Object.keys(trackers)
           .filter(username => !trackers[username].closed)
+          .slice(0, MAX_TRACKERS)
```

Comment:
Would it be better to slice at the end, so it's FIFO if we overrun the max limit?